src/hb-ot-var-fvar-table.hh

```diff
@@ -172,14 +174,14 @@ struct fvar
   find_axis_deprecated (hb_tag_t tag, unsigned *axis_index, hb_ot_var_axis_t *info) const
   {
     hb_array_t<const AxisRecord> axes = get_axes ();
-    for (unsigned i = 0; i < axes.length; i++)
-      if (axes[i].axisTag == tag)
-      {
-	if (axis_index)
-	  *axis_index = i;
-	axes[i].get_axis_deprecated (info);
-	return true;
-      }
+    unsigned i;
+    if (axes.lfind (tag, &i))
+    {
+      if (axis_index)
+	*axis_index = i;
+      axes[i].get_axis_deprecated (info);
+      return true;
+    }
```

Comment:
Another way to do this, which I typically like, is:
```
unsigned i;
if (!axis_index) axis_index = &i;
return axes.lfind (tag, axis_index) && (axes[*axis_index].get_axis_deprecated (info), true);
```

Reply:
Done